src/common.js

```diff
@@ -50,6 +84,18 @@ export function each(value, cb) {
     }
 }
 
+export function eachOwn(value, cb) {
```

Comment:
with `eachOwn`, do we still need `each`?

Reply:
Should I rename `eachOwn` to `each`, or do you like the explicitness?